webapp/src/store/events.tsx

```diff
@@ -36,6 +36,11 @@ export const docToEvent = (doc: DocumentData) => {
       id: doc.ref.id,
       start_datetime: data.start_datetime.toDate(),
       end_datetime: data.end_datetime?.toDate(),
+      // Any is used here because we literally aren't sure of the shape that's stored in the db.
+      // To address this, we will need to use a converter on the doc where we have our snapshot coming in
+      // https://firebase.google.com/docs/reference/node/firebase.firestore.FirestoreDataConverter
+      // Issue #59
```

Comment:
Added a work item to address this, but because this seems to be working alright, I'm not convinced it's actually worth doing anytime soon.